salt/modules/cmdmod.py

```diff
@@ -130,6 +130,9 @@ def _bad_level(level):
         )
         return LOG_LEVELS['info']
 
+    if salt.utils.is_true(quiet) or level.lower() == 'quiet':
```

Comment:
level can be passed in as an int, this is breaking the test. so this should be a str(level).lower() most likely